components/HeroBanner.tsx

```diff
@@ -39,7 +39,7 @@ export default function HeroBanner({
   return (
     <Box sx={overrideStyles || heroStyles} textAlign="left">
       <SectionContainer>
-        <Typography variant="h1" sx={{ display: 'block' }}>
+        <Typography variant="h1" sx={{ display: "block" }} data-testid="page-heading" >
```

Comment:
Can you please ensure that your format on save works here? Now that the settings.json file has been merged, I have a few linting errors that should be fixed on this line. If they don't, please let me know immediately! Thanks